assets/css/style.css

```diff
@@ -291,3 +318,64 @@ button:active {
 .about-myself p {
   margin-bottom: 80px;
 }
+
+.form-text {
+  margin-top: 50px;
+  width: 327px;
+  margin-left: 30px;
+  margin-bottom: 58px;
+}
```

Comment:
- [OPTIONAL] This header text in the form section is not properly aligned to the form. Kindly consider aligning it to the form as its is in the Figma design because sticking to the Figma design is a project requirement. 
your design
![qwe](https://user-images.githubusercontent.com/96683843/213239749-29445809-9147-427c-a3fb-407b643046c6.jpg)

figma design
![qwer](https://user-images.githubusercontent.com/96683843/213239712-5c1d46b2-1bc5-4133-9c26-2070d80f7f30.jpg)